docs/src/main/sphinx/connector/mariadb.rst

```diff
@@ -71,41 +71,171 @@ that catalog name instead of ``mariadb`` in the above examples.
-``varchar(n)``                      ``varchar(n)``
-``tinyblob``                        ``varbinary``
-``blob``                            ``varbinary``
-``mediumblob``                      ``varbinary``
-``longblob``                        ``varbinary``
-``varbinary(n)``                    ``varbinary``
-``date``                            ``date``
-``time(n)``                         ``time(n)``
-``timestamp(n)``                    ``timestamp(n)``                MariaDB stores the current timestamp by default.
-                                                                    Please enable `explicit_defaults_for_timestamp <https://mariadb.com/docs/reference/mdb/system-variables/explicit_defaults_for_timestamp/>`_
-                                                                    to avoid implicit default values.
-==================================  =============================== =============================================================================================================
+Because Trino and MariaDB each support types that the other does not, this
+connector modifies some types when reading or writing data. Data types may not
+map the same way in both directions between Trino and the data source. Refer to
+the following sections for type mapping in each direction.
+
+MariaDB type to Trino type mapping
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The connector maps MariaDB types to the corresponding Trino types according
+to the following table:
+
+.. list-table:: MariaDB type to Trino type mapping
+  :widths: 30, 30, 50
+  :header-rows: 1
+
+  * - MariaDB type
+    - Trino type
+    - Notes
+  * - ``BOOLEAN``
+    - ``TINYINT``
+    - ``BOOL`` and ``BOOLEAN`` are aliases of ``TINYINT(1)``
+  * - ``TINYINT``
+    - ``TINYINT``
+    -
+  * - ``SMALLINT``
+    - ``SMALLINT``
+    -
+  * - ``INT``
+    - ``INTEGER``
+    -
+  * - ``BIGINT``
+    - ``BIGINT``
+    -
+  * - ``FLOAT``
+    - ``REAL``
+    -
+  * - ``DOUBLE``
+    - ``DOUBLE``
+    -
+  * - ``DECIMAL(p,s)``
+    - ``DECIMAL(p,s)``
+    -
+  * - ``CHAR(n)``
+    - ``CHAR(n)``
+    -
+  * - ``TINYTEXT``
+    - ``VARCHAR(255)``
+    -
+  * - ``TEXT``
+    - ``VARCHAR(65535)``
+    -
+  * - ``MEDIUMTEXT``
+    - ``VARCHAR(16777215)``
+    -
+  * - ``LONGTEXT``
+    - ``VARCHAR``
+    -
+  * - ``VARCHAR(n)``
+    - ``VARCHAR(n)``
+    -
+  * - ``TINYBLOB``
+    - ``VARBINARY``
+    -
+  * - ``BLOB``
+    - ``VARBINARY``
+    -
+  * - ``MEDIUMBLOB``
+    - ``VARBINARY``
+    -
+  * - ``LONGBLOB``
+    - ``VARBINARY``
+    -
+  * - ``VARBINARY(n)``
+    - ``VARBINARY``
+    -
+  * - ``DATE``
+    - ``DATE``
+    -
+  * - ``TIME(n)``
+    - ``TIME(n)``
+    -
+  * - ``TIMESTAMP(n)``
+    - ``TIMESTAMP(n)``
+    - MariaDB stores the current timestamp by default. Enable
+      `explicit_defaults_for_timestamp <https://mariadb.com/docs/reference/mdb/system-variables/explicit_defaults_for_timestamp/>`_
+      to avoid implicit default values and create ``TIMESTAMP`` columns as
```

Comment:
and use NULL as default value.

Reply:
Added article "the" default value.

Done in commit [1d9fd79](https://github.com/trinodb/trino/pull/13656/commits/1d9fd796b945aa4f550ba142741101e5acfe4d1d)